htroisi/entity-definitions
# Add Kubernetes Job golden metrics

- FEAT: Create Pixie-MySQL entity (#826)
- FEAT: Add Pixie Postgres Entity (#828)
- Remove newRelic.ingestPoint from entity synthesis rule conditions (#834)
- Create Pixie-Cassandra entity (#835)
- Fix SNYK-UPSTREAM-NODE-3105822 (#837)
- Update Pixie-Cassandra entity definition. (#838)
- Update Pixie DNS layout (#840)
- update docs for ui changes (#836)
- Remove uninstrumented type (#842)
- Revert "Remove uninstrumented type (#842)" (#844)
- Adding mongodb prometheus exporter definitions (NR-63602) (#829)
- Fix: Remove newlines from queries behind Pixie Dashboards (#847)
- "Remove uninstrumented type (#845)
- feat: NR-10056 adding sumary metrics to Key transaction entity (#850)
- Add synthesis rules for (even) more Azure entities (#843)
- Azure host overrideGuidType (#853)
- Add the agent uid as a separate attribute (#852)
- New Zerto definition (#793)
- Use rcode name instead of rcode for Pixie DNS (#848)
- fix: Replace pixie golden metrics percentage aggregate (#846)
- Convert unit of 2 goldnen metrix from ms to second to match real data (#839)
- Update fortinet-fortigate-dashboard.json (#833)
- Run Sanitize dashboards in repo (#827)
- Entity Synthesis for InfluxDB (#783)
- Add ext-net_snmp (#805)
- FEAT: Pixie-AMQP entity definition (#849)
- fix: change max to latest for CPU widget (#860)
- Fix: Pixie Golden Metrics order of operations (#858)
- add synthesis rule for Azure CosmosDB account (#857)
- Change Azure host identifier (#856)
- Add ruckus wlc (#855)
- NEWRELIC-4198 : Update OTEL Host golden metrics (network.io) (#862)
- Add new default dashboard templates for routers and switches (#854)
- feat: add shelled_spa (#809)
- Add jpujol test entity type for onboarding exercise (#863)
- feat: add golden metrics to ext-service_level (#851)
- Feat/add golden metrics to ext service level (#864)
- Create separate default firewall dashboard template (#865)
- Feat/add golden metrics to ext service level (#866)
- resourcetypes to lowercase (#867)
- Revert "NEWRELIC-4198 : Update OTEL Host golden metrics (network.io) (#862)" (#868)
- Azure resourceType lowercase (#870)
- Add cluster name to Pixie entities summary metrics (#869)
- Azure resourceType lowercase 3 (#871)
- new entity type: ext-dell_datadomain  (#861)
- fix: ext-waf dashboard (#874)
- Fix: ext-trap_device. golden tag and dashboard template (#872)
- Avoid Cloud metrics showing empty agent Hosts (#876)
- Fix:fortigate firewall cpu changes (#873)
- New Type: INFRA-AWSFSX volumes (#877)
- fix: NR-10056 modified the query for summary metrics for KT throguput (#878)
- Update JSON libs to fix vulnerabilities (#881)
- Net snmp add esx (#883)
- Validate that sanitize dashboards was run (#888)
- Ensure that off-cluster Pixie entities are properly captured (#884)
- NEWRELIC-5898: Add breakdown of Redis metrics based on responding pods and requesting pods (#886)
- NEWRELIC-5898 Postgres split by requesting pod and split by responding pod (#889)
- NEWRELIC-5898 Pixie Kafka split by requesting pod and split by responding pod (#890)
- NEWRELIC-5898 Pixie MySQL split by requesting pod and split by responding pod (#891)
- NEWRELIC-5898 Pixie Cassandra Dashboard updates: split by requesting pod and split by responding pod (#894)
- adding build artifact type (#887)
- NEWRELIC-5898 Pixie AMQP Dashboard updates: split by requesting pod and split by responding pod (#892)
- NEWRELIC-5898 Pixie DNS Dashboard updates: split by requesting pod and split by responding pod (#893)
- EXT SolarWinds Servers (#882)
- [NR-78597] chore: update node library to avoid security issue (#899)
- update Azure definitions to lower-case (#900)
- Introduce uninstrumented hosts and containers (#902)
- Sanitize dashboards command removes empty variable arrays (#898)
- Make viz-dashboard entity TTL MANUAL (#895)
- Update definition.yml (#879)
- Aerospike - New Entity Definition (#904)
- Use `NA` instead of `entity.type` for Azure entities (#907)
- Update golden_metrics.yml (#908)
- feat: add workflow entity definition (#909)
- Update definition.yml (#910)
- feat: add avtech roomalert to ext_environment-sensor (#896)
- Display redis error metrics for Pixie redis entity (#897)
- Mongodb gold and summary metrics (#901)
- Cleanup test types (#905)
- Remove opamp.agent.uid tag taken from instance.id (#906)
- Correct typo (#911)
- Update CONTRIBUTING.md (#915)
- Add SSL Certifcate Entity Definition (#903)
- Update definition.yml (#912)
- Mongodb dashboards (#917)
- Fix golden metrics for azureappserviceplan (#918)
- feat: create jfrog entity (#916)
- Add support for filterNRQLByEntityID in definition.yml (#920)
- Add Kubernetes Job, CronJob entities (#921)
- NR-81315 Added event name to azureappserviceplan (#923)
- Revert "Add Kubernetes Job, CronJob entities (#921)" (#924)
- Revert "Revert "Add Kubernetes Job, CronJob entities (#921)" (#924)" (#925)
- Gigamon dns (#928)
- Revert "Add Kubernetes Job, CronJob entities (#921)" (#932)
- new definition: ext-billable_user (#919)
- Fix: typo in gigamon definition (#933)
- aerospike - update on entity definition (#931)
- Enable ext-billable_user (#934)
- node upgrade by snyk recomendation (#929)
- Expire host & containers uninstrumented (#926)
- make entities non-alertable (#936)
- Add Kubernetes Job, CronJob entities (#941)
- Add new Kubernetes entities: PV, PVC, Endpoint, ReplicaSet (#942)
- Introduce uninstrumented S3, SQS, SNS (#944)
- Add squid exporter otel definition (#950)
- Add snmp appliance (#946)
- Adding request duration metric with histogrampercentile (#939)
- feat: Change title to reflect query (#948)
- Revert "Adding request duration metric with histogrampercentile (#939)" (#951)
- Add lujop to CODEOWNERS (#952)
- feat: create synthetics runtime upgrade validation entity (#945)
- Update Gigamon DNS Service entity (#947)
- Remove unused Kubernetes dashboards (#955)
- Add Kubernetes CronJob dashboard (#954)
- Add Site (#938)
- ext-service: Add deployment tag for data that comes from Pixie (#959)
- Fix EXT_SITE summary metrics (#961)
- update jfrog entity (#953)
- Simplified queries and removed host_type (#963)
- Add tag for definition (#965)
- chore: adding supported nrql syntax to gm docs (#958)
- Dell Avamar synthesis init (#943)
- Entity definitions for Azure Batch (#967)
- fix: correct where conditions (#970)
- billable_user: update definition to composite (#971)
- feat: Policies as entities (#972)
- New INFRA types: AWSTEXTRACT & AWSTRANSLATE (#935)
- INFRA new type AWSLEX (#968)
- INFRA Squid Cache Manager improve experience (GM, SM & dashboards) (#975)
- Feat: Add new ext-hardware_security_module (#969)
- ext-network_dns_service: Updated the instrumentation.name (#976)
- EXT-FIREWALL: Update cisco-asa-dashboard.json (#977)
- Validator fixes on equality rule check and tone it down logs (#980)
- Add Kubernetes Job golden metrics

### Relevant information

Describe what you have done and any details that you think are relevant or that you might want to discuss with us. 

### Checklist

* [ ] I've read the guidelines and understand the acceptance criteria.
* [ ] The value of the attribute marked as `identifier` will be unique and valid. 
* [ ] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.
